README.md

```diff
@@ -18,4 +18,16 @@ Then you need to run mkdocs from the root of that repository:
 
     $ mkdocs build
 
+Sometimes this might fail on Fedora 24+, so perform below steps
```

Comment:
@prashanthpai , I retried it. It is still failing with the below error
"
ERROR   -  Config value: 'docs_dir'. Error: The 'docs_dir' should not be the parent directory of the config file. Use a child directory instead so that the config file is a sibling of the config file. 
ERROR   -  Config value: 'site_dir'. Error: The 'site_dir' should not be within the 'docs_dir' as this leads to the build directory being copied into itself and duplicate nested files in the 'site_dir'.(site_dir: '/home/psony/glusterdocs/site', docs_dir: '/home/psony/glusterdocs') 
Aborted with 2 Configuration Errors!
"